scripts/run-unit-tests.ps1

```diff
@@ -8,22 +8,21 @@ Param(
 
 $LIBSODIUM_GIT_URL = "https://github.com/jedisct1/libsodium"
 
-# $GOLANG_DOWNLOAD_LINK = "https://dl.google.com/go/go1.9.4.windows-amd64.msi"
 $GOLANG_DOWNLOAD_LINK = "https://dl.google.com/go/go1.11.windows-amd64.msi"
 $GIT_DOWNLOAD_LINK = "https://github.com/git-for-windows/git/releases/download/v2.16.1.windows.3/Git-2.16.1.3-64-bit.exe"
 $7_ZIP_DOWNLOAD_LINK = "https://www.7-zip.org/a/7z1801-x64.exe"
 $2012_RUNTIME_DOWNLOAD_LINK = "https://download.microsoft.com/download/1/6/B/16B06F60-3B20-4FF2-B699-5E9B7962F9AE/VSU_4/vcredist_x64.exe"
 $2013_RUNTIME_DOWNLOAD_LINK = "https://download.microsoft.com/download/2/E/6/2E61CFA4-993B-4DD4-91DA-3737CD5CD6E3/vcredist_x64.exe"
 $DIG_DOWNLOAD_LINK = "http://dcos-win.westus2.cloudapp.azure.com/downloads/dig-x64.zip"
-$OTP202_DOWNLOAD_LINK = "http://dcos-win.westus2.cloudapp.azure.com/downloads/erl9.2.zip"
+$OTP_DOWNLOAD_LINK = "http://erlang.org/download/otp_win64_21.1.exe"
```

Comment:
Just calling out that we're moving from a controlled repository to erlang's download page directly.